seleniumByTechnocredits/src/seleniumPractise/vrushali/JavascriptConfirmation.java

```diff
@@ -0,0 +1,71 @@
+//javaScript Confirmation> Ok/Cancel > Validate the message shown below button with OK/Cancel action.
+
+package seleniumPractise.vrushali;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import util.Init;
+
+public class JavascriptConfirmation {
+
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		JavascriptConfirmation.JavascriptConfirmationOK(driver);
+		JavascriptConfirmation.JavascriptConfirmationCancel(driver);
+		driver.close();
+
+	}
+
+	// Method to validate message after accepting (OK) javaScript Confirmation
+	// alert
+	public static void JavascriptConfirmationOK(WebDriver driver) throws InterruptedException {
+
+		driver.get("http://automationbykrishna.com/#");
+		driver.findElement(By.xpath("//a[@id= 'basicelements']")).click();
+		System.out.println("Basic element tab clicked");
+		driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
```

Comment:
i think we should avoid using implicit wait. As it will add unnecessary polling time (250ms)before every line of code.

Reply:
@VrushaliPTechnocredits
For JavascriptConfirmation,  Code looks good to me.
1. In  main method, instead driver.close() you can use driver.quit(), this will close 2 browsers which you are opening instead of closing only 1 browser via driver.close().
2. Below lines of code are repeated in both methods. driver.get(url) can be written in main to avoid opening browser twice.
driver.get("http://automationbykrishna.com/#");
		driver.findElement(By.xpath("//a[@id= 'basicelements']")).click();
		System.out.println("Basic element tab clicked");
		driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);

Reply:
Above comments are incorporated.